frontend/luasettings.lua

```diff
@@ -60,6 +60,10 @@ function LuaSettings:flipTrue(key)
     end
 end
 
+function LuaSettings:replace(table)
```

Comment:
I will rename it to reset() to match std::unique_ptr<>. Though IMO, it's not a good name. Following statements look correct to me.
`object.reset()`
`object.replace(other)  // This one is a little bit confusing. It's a little bit unclear whether the object is replaced by other, or object replaces other.`
`object.replace_with(other)`
`object.is_replaced_by(other)  // This one should be better than the one above, but it looks pretty long.`

But
`object.reset(other)`
is even more confusing.